File: app/src/main/java/com/bogdanorzea/popularmovies/model/Genre.java
package com.bogdanorzea.popularmovies.model;

import com.squareup.moshi.Json;

public class Genre {

    @Json(name = "id")
    public int id;
    @Json(name = "name")
    public String name;

    public Genre withId(int id) {
        this.id = id;
        return this;
    }

    public Genre withName(String name) {
        this.name = name;
        return this;
    }
}

File: app/src/main/java/com/bogdanorzea/popularmovies/model/object/Genre.kt
package com.bogdanorzea.popularmovies.model.`object`

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Genre(@Json(name = "id") var id: Int,
                 @Json(name = "name") var name: String) : Parcelable {

    override fun toString(): String = name

}
